src/resources/js/script.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file This file contains all JavaScript specific to Schedule Day View.
+ * This file should load after all vendor and core events JavaScript.
+ * @version 1.0.0
+ */
+
+(function ( window, document, $, td, te, tf, ts, tt, config, dbug ) {
+
+	/*
+	 * $    = jQuery
+	 * td   = tribe_ev.data
+	 * te   = tribe_ev.events
+	 * tf   = tribe_ev.fn
+	 * ts   = tribe_ev.state
+	 * tt   = tribe_ev.tests
+	 * dbug = tribe_debug
+	 */
+
+	// @ifdef DEBUG
+	if ( dbug ) {
+
+	}
+	// @endif
+
+	$( document ).ready( function () {
+		var $container = $( '.tribe_ext_sch_day_view #tribe-events-day' ),
+			tribe_timezone = $container.data( 'tribe-timezone' ),
```

Comment:
No need to change this, but worth noting for future ref that the preferred practice is to use [multiple var statements](http://moderntribe.github.io/products-engineering/code-standards/javascript/#variable-declaration) rather than a single var that declares multiple variables.